lib/visitor/compiler.js

```diff
@@ -413,8 +413,8 @@ Compiler.prototype.visitUnit = function(unit){
 
   // Compress
   if (this.compress) {
-    // Always return '0' unless the unit is a percentage or time
-    if ('%' != type && 's' != type && 'ms' != type && 0 == n) return '0';
+    // Always return '0' unless the unit is a percentage, time, degree or fraction
+    if ('%' != type && 's' != type && 'ms' != type && 'deg' != type && 'fr' != type && 0 == n) return '0';
```

Comment:
Too many check in a single IfStatement, can you refact it to more clean ? and unit test is also required! Thanks for contribution !

Reply:
I've now made this statement more concise, and added the unit tests! Let me know if anything else needs to be done.